### PR TITLE
Expose necessary attributes to do streaming replication, bump version

### PR DESCRIPTION
**Pull request by igueths**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name             'pgpool'
-maintainer       'Jens Jorritsma'
-maintainer_email 'jens.jorritsma@gamil.com'
+maintainer 'Jens Jorritsma'
+maintainer_email 'jens.jorritsma@rackspace.com'
```
Comment (jmvrbanac) [inline metadata.rb]: Unless the maintainer requests his/her email to be changed, lets not change this.

File: attributes/default.rb [hunk 2]
```diff
@@ -3,17 +3,20 @@
 
 case node['platform']
 when 'debian', 'ubuntu'
-  default['pgpool']['config']['package_name'] = 'pgpool2'
+  default['pgpool']['config']['package_name'] = 'pgpool-II-pg93' # Might override
```
Comment (jmvrbanac) [inline attributes/default.rb]: This is for debian and ubuntu and I'm pretty sure the package name is pgpool2

File: attributes/default.rb [hunk 3]
```diff
@@ -3,17 +3,20 @@
...
+# increment the default here such that it installs the version meant for
+# Postgres-9.3.x, regardless this is overrideable like any other attribute.
+default['pgpool']['config']['package_name'] = 'pgpool-II-pg93'
   default['pgpool']['config']['dir'] = '/etc/pgpool-II'
   default['pgpool']['service'] = 'pgpool'
 end
 
 default['pgpool']['pgconf']['listen_addresses'] = 'localhost'
-default['pgpool']['pgconf']['port'] = 5432
+default['pgpool']['pgconf']['port'] = 9999
```
Comment (jmvrbanac) [inline attributes/default.rb:19]: Why the port change?

File: attributes/default.rb [hunk 4]
```diff
@@ -24,19 +27,99 @@
...
+default['pgpool']['pgconf']['backend_port1'] = 5432
+default['pgpool']['pgconf']['ssl'] = false # Should override if SSL is desired.
+default['pgpool']['pgconf']['ssl_key'] = '' # Should override
+default['pgpool']['pgconf']['ssl_cert'] = '' # Should override
+default['pgpool']['pgconf']['ssl_ca_cert'] = '' # Should override
+default['pgpool']['pgconf']['ssl_ca_cert_dir'] = '' # Should override
 default['pgpool']['pgconf']['replication_mode'] = false
-default['pgpool']['pgconf']['load_balance_mode'] = false
-
+default['pgpool']['pgconf']['load_balance_mode'] = true
```
Comment (jmvrbanac) [inline attributes/default.rb:44]: Should this be set to true by default?

File: README.md [hunk 5]
```diff
@@ -30,7 +30,193 @@ Attributes
     <td>log directory</td>
     <td><tt>/var/log/pgpool</tt></td>
   </tr>
+<tr>
```
Comment (jmvrbanac) [inline README.md:33]: Is it possible to switch this over to a markdown table instead? It'll be a lot easier to read and review.
https://help.github.com/articles/github-flavored-markdown/#tables

File: attributes/default.rb (hunk 3 above)
Comment (igueths) [inline attributes/default.rb:19]: Given that these are Postgres clients that would be connecting, I saw no reason to leave that particular default at 9999 given that running a DB plus Pgpool on the same machine isn't exactly encouraged.

File: attributes/default.rb (hunk 4 above)
Comment (igueths) [inline attributes/default.rb:44]: It should if one desires that non-DDL/DML queries be transparently spread across any slaves that may be available, which is usually the case.

File: README.md (hunk 5 above)
Comment (igueths) [inline README.md:33]: I can drop an issue against the readme to get it reworked from its original format, for the purposes of this PR though imo this particular item can be pushed to a future iteration.

File: attributes/default.rb (hunk 2 above)
Comment (jfwood) [inline attributes/default.rb]: I'm presuming the note below applies here as well, to ensure a 9.3 pgpool is pulled?

Comment (jmvrbanac) [inline attributes/default.rb]: The centos package name should be fine, but the debian and ubuntu package name isn't valid and will fail install.

File: attributes/default.rb (hunk 4 above)
Comment (jfwood) [inline attributes/default.rb:44]: I agree if there is no other reason for us to use pgpool other than as a LB and read-only/read-write connection handler, which I think is the case.

File: templates/default/pgpool.conf.erb [hunk 6]
```diff
@@ -1,3 +1,4 @@
+# This file was autogenerated by Chef, do not edit!
```
Comment (jfwood) [inline templates/default/pgpool.conf.erb:1]: Sooo...does adding this comment count as an edit? :)

Comment (igueths) [inline templates/default/pgpool.conf.erb:1]: Hahaha not that comment.

**Comment by jfwood** [comment]
+1

**Comment by jstraw** [comment]
+1

**Pull request merged**